graphql/schemabuilder/pagination.go

```diff
@@ -606,12 +604,8 @@ func (c *connectionContext) extractPaginatedRetAndErr(out []reflect.Value, args
 	if err != nil {
 		return nil, err
 	}
-	out = out[1:]
-	if c.ReturnsPageInfo {
```

Comment:
presumably unnecessary because callers should be in this codepath?

Reply:
Yep